Doc/c-api/sys.rst

```diff
@@ -222,10 +222,55 @@ These are utility functions that make functionality from the :mod:`sys` module
 accessible to C code.  They all work with the current interpreter thread's
 :mod:`sys` module's dict, which is contained in the internal thread state structure.
 
+.. c:function:: PyObject *PySys_GetAttr(PyObject *name)
+
+   Return the object *name* from the :mod:`sys` module or ``NULL`` on failure.
+   Set :exc:`RuntimeError` and return ``NULL`` if it does not exist.
```

Comment:
RuntimeError is a little bit surprising. Why not AttributeError?

Does the function return a borrowed reference or a strong reference? Please make it explicit.

I suggest:

```suggestion
   Get the attribute *name* of the :mod:`sys` module. Return a :term:`strong reference`.
   Raise :exc:`RuntimeError` and return ``NULL`` if it does not exist.
```

Reply:
> RuntimeError is a little bit surprising. Why not AttributeError?

Because it was always raised if `PySys_GetObject()` or variants did not find an object. I just moved it inside the function. Do you prefer to repeat code like `PyErr_SetString(PyExc_RuntimeError, "lost sys.stdin");` at caller place?

> Does the function return a borrowed reference or a strong reference? Please make it explicit.

It returns a strong reference like all other C API, unless opposite is specified.